syft/workers/base.py

```diff
@@ -583,7 +599,7 @@ def get_worker(self, id_or_worker, fail_hard=False):
 
         return id_or_worker
 
-    def add_worker(self, worker):
+    def add_worker(self, worker: "BaseWorker"):
```

Comment:
Why is this a string?

Reply:
You can mention a type if you are in its own class, that's kind the way of handling it

Reply:
Ah okay